manifests/web-cluster/sdlc/helium/account-postgres-sink-service.yaml

```diff
@@ -187,29 +187,3 @@ spec:
             name: account-postgres-sink-service
             port:
               number: 8081
----
-apiVersion: autoscaling/v2
-kind: HorizontalPodAutoscaler
-metadata:
-  name: account-postgres-sink-service
-  namespace: helium
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: account-postgres-sink-service
-  minReplicas: 1
-  maxReplicas: 1
-  metrics:
-    - type: Resource
-      resource:
-        name: cpu
-        target:
-          type: Utilization
-          averageUtilization: 75
-    - type: Resource
-      resource:
-        name: memory
-        target:
-          type: Utilization
-          averageUtilization: 75
```

Comment:
This won't do anything since `minReplicas: 1`, `maxReplicas: 1`. If we want HPA gotta change `maxReplicas` to something >1. Deleted for the meantime.